{
  try {
    parent.processingInstruction("foo","bar");
  }
 catch (  SAXException e) {
    throw new RuntimeException("Unexpected exception",e);
  }
  assertEquals(logger.size(),1);
  assertEquals("processingInstruction",logger.getMethod());
  assertEquals(new Object[]{"foo","bar"},logger.getArgs());
}
